modules/node_modules/@ncigdc/components/Modals/FirstTimeModal.js

```diff
@@ -13,6 +13,7 @@ const FirstTimeModal = () => (
     <p>Unauthorized attempts to upload information, change information, or use of this web site may result in disciplinary action, civil, and/or criminal penalties. Unauthorized users of this web site should have no expectation of privacy regarding any communications or data processed by this web site.</p>
     <p>Anyone accessing this web site expressly consents to monitoring of their actions and all communication or data transiting or stored on or related to this web site and is advised that if such monitoring reveals possible evidence of criminal activity, NIH may provide that evidence to law enforcement officials.</p>
     <p>Please be advised that some features may not work with higher privacy settings, such as disabling cookies.</p>
+    <p>WARNING: Please read the <ExternalLink hasExternalIcon={false} href="https://docs.gdc.cancer.gov/Data/Release_Notes/Data_Release_Notes">GDC Data Release Notes</ExternalLink> prior to accessing this web site as the Release Notes provide details about data updates, and known issues and workarounds.</p>
```

Comment:
> updates, and known issues and workarounds.

reads a little awkward. could that be "updates, known issues and workarounds." ?